source/extensions/filters/http/ip_tagging/ip_tagging_filter.cc

```diff
@@ -19,8 +19,8 @@ IpTaggingFilterConfig::IpTaggingFilterConfig(
     : request_type_(requestTypeEnum(config.request_type())), scope_(scope), runtime_(runtime),
       stat_name_set_(scope.symbolTable().makeSet("IpTagging")),
       stats_prefix_(stat_name_set_->add(stat_prefix + "ip_tagging")),
-      hit_(stat_name_set_->add("hit")), no_hit_(stat_name_set_->add("no_hit")),
-      total_(stat_name_set_->add("total")) {
+      no_hit_(stat_name_set_->add("no_hit")), total_(stat_name_set_->add("total")),
+      unknown_tag_(stat_name_set_->add("unknown_tag.hit")) {
```

Comment:
I had thought what we could do was walk through the tags and add all of them as builtins so we don't map them all to `unknown`. Was there some reason you didn't want to do it that way?

Reply:
Eh?

I am doing that, in the loop that discovers the tags... the `unknown_tag_` is needed because `getBuiltin()` needs a fallback parameter...